pkg/daemon/certificate_writer.go

```diff
@@ -183,6 +198,24 @@ func (dn *Daemon) syncControllerConfigHandler(key string) error {
 			return fmt.Errorf("failed to set ControllerConfigResourceVersion annotation on node: %w", err)
 		}
 		klog.Infof("Certificate was synced from controllerconfig resourceVersion %s", controllerConfig.ObjectMeta.ResourceVersion)
+
+		if len(controllerConfig.Spec.ImageRegistryBundleUserData) > 0 || newCloudCA {
+			cmd := exec.Command("update-ca-trust")
+			var stderr bytes.Buffer
+			cmd.Stdout = os.Stdout
+			cmd.Stderr = &stderr
+			if err := cmd.Run(); err != nil {
+				return fmt.Errorf("error running update-ca-trust: %s: %w", string(stderr.Bytes()), err)
+			}
+
+			serviceName := "crio"
+			if err := restartService(serviceName); err != nil {
+				if dn.nodeWriter != nil {
+					dn.nodeWriter.Eventf(corev1.EventTypeWarning, "FailedServiceReload", fmt.Sprintf("Reloading %s service failed. Error: %v", serviceName, err))
+				}
+				return fmt.Errorf("could not apply update: reloading %s configuration failed. Error: %w", serviceName, err)
+			}
```

Comment:
Let's add a log message about successful crio restart and update-ca-cert

Reply:
I can see crio restart log in daemon log